### DISCUSSION TITLE
Report a bug in elementor

### DISCUSSION THREAD

**Discussion by 79ho3ein** [elementor/elementor#14419] [Developers Community]
hello

In the Safari browser, the text of the text turns red when we dynamically associate the icon list widget. Not so in Android browsers.
Please check this item.
thanks

**Comment by d19dotca** [comment]
You should actually create an Issue rather than a Discussion in GitHub here for the Elementor repository, where you can also share more details such as how to reproduce the issue, what your setup is and what plugins are used, which versions, etc.

**Reply by 79ho3ein** [reply]
Can you explain how to make it?

**Reply by d19dotca** [reply]
You simply go to Issues > New Issue > Bug Report. The direct link to creating a new bug report is here: https://github.com/elementor/elementor/issues/new?assignees=&labels=&template=Bug_report.md
